### ISSUE TITLE
Can we have an arm build

### ISSUE THREAD

**Issue by jengo** [munnerz/kube-plex#19]
I was going to do it my self, but the build instructions are missing and I am not fully sure exactly how you build this project.  Ideally, it would be best to provide a multi-architecture docker image.

**Comment by archite** [comment]
```
CGO_ENABLED=0 GOOS=linux GOOARCH=arm go build -a -installsuffix cgo -o kube-plex_linux_arm github.com/munnerz/kube-plex
sed -e 's@alpine:3.6@arm32v6/alpine@' -e 's@kube-plex_linux_amd64@kube-plex_linux_arm@' Dockerfile | docker build -t munnerz/kube-plex-arm -f - .
```

**Comment by mauroniewolskicesca** [comment]
I forked the project, modified it and now using it on my arm64 device https://github.com/mauroniewolskicesca/kube-plex-arm64

**Comment by gclawes** [comment]
With the Raspberry Pi 4 out it'd be great to have an official arm64 image.

**Comment by JoseThen** [comment]
> I forked the project, modified it and now using it on my arm64 device https://github.com/mauroniewolskicesca/kube-plex-arm64

Hey @mauroniewolskicesca , I know its a little late but how did you modify it ? Might be a good idea to use those steps and set up a pr for this repo so others can use.

**Comment by jantznick** [comment]
Is there any thought of doing this? I thought I saw somewhere before you can just update the download file but I don't remember if that was this build or a different one and can't find it again.

**Comment by JoseThen** [comment]
Using @archite 's comment I was able to build for ARM, I'll post here later this week more info, wouldn't be a bad PR for this repo!

**Comment by jantznick** [comment]
@JoseThen Wasn't understanding that comment at first, so it basically replaces the version of plex being pulled in the Docker file?

**Comment by JoseThen** [comment]
> @JoseThen Wasn't understanding that comment at first, so it basically replaces the version of plex being pulled in the Docker file?

Yes, it first does a different arch build with Go and changes the image name in the dockerfile, I have time I'll post again later today

**Comment by JoseThen** [comment]
For those looking I forked this repositoryand switched it to build for ARM, you can see the main diff here: https://github.com/munnerz/kube-plex/compare/master...JoseThen:master

I have it set to push to my own docker registry. I recommend building this out on your own, because its a great learning process.

**Comment by o4fr** [comment]
> For those looking I forked this repositoryand switched it to build for ARM, you can see the main diff here: [master...JoseThen:master](https://github.com/munnerz/kube-plex/compare/master...JoseThen:master)
> 
> I have it set to push to my own docker registry. I recommend building this out on your own, because its a great learning process.

Great work @JoseThen. Does it also work for arm64 e.g. Raspberry Pi 4?

**Comment by gclawes** [comment]
Any update on this?
